src/main/java/com/ibm/cloud/sdk/core/service/BaseService.java

```diff
@@ -435,10 +447,17 @@ public void onResponse(Call call, Response response) {
     public Single<com.ibm.cloud.sdk.core.http.Response<T>> reactiveRequest() {
       return Single.fromCallable(new Callable<com.ibm.cloud.sdk.core.http.Response<T>>() {
         @Override
-        public com.ibm.cloud.sdk.core.http.Response<T> call() throws Exception {
-          Response response = call.execute();
-          T responseModel = processServiceCall(converter, response);
-          return new com.ibm.cloud.sdk.core.http.Response<>(responseModel, response);
+        public com.ibm.cloud.sdk.core.http.Response<T> call() {
+          try {
+            Response response = call.execute();
+            T responseModel = processServiceCall(converter, response);
+            return new com.ibm.cloud.sdk.core.http.Response<>(responseModel, response);
+          } catch (IOException e) {
```

Comment:
Do we truly want to catch and re-throw only IOExceptions as a RuntimeException?   Since you removed the throws class from the function signature, shouldn't we catch all exceptions (i.e. Throwable) and then re-throw as a RuntimeException?

Reply:
`enqueue` needs an `IOException` in the overridden signature for `onFailure`, but I went ahead and changed it to throw all `Exception`s for `execute` and `reactiveRequest`.

Reply:
Actually, I think we should stick with just throwing `IOException`s. The `processServiceCall` method will handle returning the more specific exceptions like 404s and 500s. With the change I just made, tests that are looking for more specific errors are breaking.

Reply:
Ok, do what you need to do.   I added my original comment because I noticed the potential change in behavior is all.